src/popup/components/NotificationContainer.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+
+import ButtonIcon from './ButtonIcon'
+import styles from './Button.css'
+import * as constants from '../constants'
+import { remoteFunction } from '../../util/webextensionRPC'
+
+const getUnreadCount = remoteFunction('getUnreadCount')
+
+class NotificationContainer extends Component {
+    state = {
+        unreadNotifCount: 0,
```

Comment:
Doesn't the unread notif count state initialization happen inside your `init` thunk in `notifications/actions`? Is there a need for this container?

Reply:
> Is there a need for this container?

Here we are showing the # of notifs in the popup too.